nfsstat/tests/compose/compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+
+services:
+  nfs-server:
+    image: gists/nfs-server
+    container_name: nfs-server
+    privileged: true
```

Comment:
Maybe let's add a comment about why we need such a flag - I think that's because we need `CAP_SYS_ADMIN` to be able to `mount` filesystems.

(Btw have we re-checked that `cap_add: sys_admin` doesn't do the trick instead of the `privileged` flag?)

Reply:
👍 for the doc

And actually, `cap_add: sys_admin` is not enough for the test to run on Azure (that's why the CI failed before)